NLibrary/Classes/PlaylistMVVM/PlaylistItemBasicModel.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  UserProtocol.swift
+//  taylorswift
+//
+//  Created by Nathan Ngobale on 2020/03/20.
+//  Copyright © 2020 Nathan Ngobale. All rights reserved.
+//
+
+import Foundation
+import AVKit
+
+public struct PlaylistBasicModel {
+    public var playlistItems: [PlaylistItemBasicModel] = []
+    public var authorId: String?
+    init?(dict: NSDictionary) {
+        guard let keyValues = dict.allValues.first as? NSDictionary else {
+            return
+        }
+        guard let authorId = dict.allKeys.first as? String else {
+            return
+        }
+        self.authorId = authorId
+        for (key, value) in keyValues {
+            guard let key = key as? String,
+                let page = value as? NSDictionary else {
+                return
+            }
+            playlistItems.append(PlaylistItemBasicModel(albumName: key, dictionary: page))
+        }
+    }
+    
+    var dict: [String : Any]?
+}
+
+public struct PlaylistItemBasicModel: DictionaryConvertible {
+    public var id: String = ""
+    public var name: String = ""
+    public var numSongs: Int = 0
+    public var basicSongItems: [BasicSongItem] = []
+    public var dict: [String: Any] {
+        return [
+            "name": name,
+            "numSongs": numSongs
+        ]
+    }
+
+    init() {
+    }
+
+    init?(dict: [String: Any]) {
+        self.init()
+    }
+
+    init(albumName: String, dictionary: NSDictionary) {
+        name = albumName
+        let numSongs = dictionary.allValues.count
+        self.numSongs = numSongs
+        guard let songId = dictionary.allKeys.first as? String else {
+            return
+        }
+        for page in dictionary.allValues {
+            let page = page as! [String: Any]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Force casts should be avoided.](https://app.codacy.com/manual/NathanDVT/NLibrary/pullRequest?prid=5409525)